db/config.cc

```diff
@@ -1576,7 +1576,8 @@ std::map<sstring, db::experimental_features_t::feature> db::experimental_feature
         {"broadcast-tables", feature::BROADCAST_TABLES},
         {"keyspace-storage-options", feature::KEYSPACE_STORAGE_OPTIONS},
         {"tablets", feature::UNUSED},
-        {"views-with-tablets", feature::VIEWS_WITH_TABLETS}
+        {"views-with-tablets", feature::VIEWS_WITH_TABLETS},
+        {"rf-rack-restricted-keyspaces", feature::RF_RACK_RESTRICTED_KEYSPACES}
```

Comment:
This is bikeshedding, but I'd argue that rf-rack-matched-keyspaces is more descriptive.